packages/riipen-ui/src/components/TableContainer.jsx

```diff
@@ -0,0 +1,80 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+
+import ThemeContext from "../styles/ThemeContext";
+
+class TableContainer extends React.Component {
+  static propTypes = {
+    /**
+     * Children to render in the table element
+     */
+    children: PropTypes.node,
+
+    /**
+     * Classes to apply to the root element
+     */
+    classes: PropTypes.arrayOf(PropTypes.string),
+
+    /**
+     * Whether to give all columns in the table equal width
+     */
+    fixed: PropTypes.bool,
```

Comment:
I think this should be renamed to `tableLayout` and should have the whitelisted options available in the CSS.